apps/crn-server/test/data-providers/contentful/users.data-provider.test.ts

```diff
@@ -680,6 +694,32 @@ describe('User data provider', () => {
           }),
         );
       });
+      test('Should query data properly when passing search param', async () => {
+        contentfulGraphqlClientMock.request.mockResolvedValueOnce(
+          getContentfulGraphqlUser(),
+        );
+
+        const search = 'Tag';
+        await userDataProvider.fetch({
+          search,
+        });
+
+        expect(contentfulGraphqlClientMock.request).toHaveBeenCalledWith(
+          expect.anything(),
+          expect.objectContaining({
+            limit: 8,
+            order: ['lastName_ASC'],
+            skip: 0,
+            where: {
+              AND: [
+                { OR: [{ expertiseAndResourceDescription_contains: 'Tag' }] },
```

Comment:
I don't understand this one, aren't we removing `expertiseAndResources` field and switching to research-tags?

Reply:
It's for the script to be working. Once everything is migrated we would be able to remove the whole field (including that search).